docs/Recipes.md

```diff
@@ -163,3 +163,39 @@ Set the environment variable `APP_ENV` to `detox_tests` when running Haul:
 ```sh
 APP_ENV=detox_tests yarn haul
 ```
+
+## Debugging with React Native Tools (`vscode-react-native`)
+
+In order to use React Native Tools extension you must first install the extension:
+```bash
+code --install-extension msjsdiag.vscode-react-native
+```
+
+Next, you need to tweak your `haul.config.js` and add `source-map-loader`:
+```diff
+import { withPolyfills, makeConfig } from "@haul-bundler/preset-0.60";
+
+export default makeConfig({
+  bundles: {
+    index: {
+      entry: withPolyfills('./index.js'),
++      transform({ config }) {
```

Comment:
Other examples in this file don't use the diff format (with `+` at the start), which breaks syntax highlighting and makes it harder to copy/paste.